pkg/machine/hyperv/volumes.go

```diff
@@ -48,7 +48,7 @@ func startShares(mc *vmconfigs.MachineConfig) error {
 		if requiresChattr {
 			args = append(args, "sudo", "chattr", "-i", "/", "; ")
 		}
-		args = append(args, "sudo", "mkdir", "-p", cleanTarget, "; ")
+		args = append(args, "sudo", "mkdir", "-p", fmt.Sprintf("\"%s\"", cleanTarget), "; ")
```

Comment:
just use `%q` instead that will also escape more special chars.

Reply:
Better yet, use `strconv.Quote`.

Reply:
Fixed. Thanks.